### DISCUSSION TITLE
Fast Forward button?

### DISCUSSION THREAD

**Discussion by Blitzoreo** [visualboyadvance-m/visualboyadvance-m#1356] [General]
I'm very sure I remember VBA having a Fast Forward button, it was set to the "Spacebar" by default, dunno if it was even configurable (from all the poor souls who don't like it, seems it's not).

I can't seem to find it anywhere in the latest VisualBoyAdvance-M 2.1.0

**Comment by ZachBacon** [comment]
Check your key configuration in the input settings.

**Comment by Blitzoreo** [comment]
I see, so it's on "Options>Input>Configure...", not on the "Key Shortcuts..." menu as I thought.

Thanks.

**Comment by ZachBacon** [comment]
There maybe an issue with the inputs, I'll leave this open for now and let me know if you have issues

**Comment by pianistrevor** [comment]
Should be under Options > Input > Configure... > Special (Tab) > Speed Up.
